docs/topics/about-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
```

Comment:
Shouldn't this module be deleted if you are replacing it with a common 'about plugin' module?